golem-base-indexer/golem-base-indexer-logic/src/repository/sql.rs

```diff
@@ -276,3 +276,17 @@ ORDER BY
     entities_created_count DESC,
     first_created_at ASC
 "#;
+
+pub const GET_ADDRESS_ACTIVITY: &str = r#"
+SELECT
+    MIN(transactions.block_timestamp) AS first_seen,
+    MAX(transactions.block_timestamp) AS last_seen
+FROM
+    transactions
+WHERE
+    transactions.block_timestamp IS NOT NULL
+    AND (
+        transactions.from_address_hash = $1
+        OR transactions.to_address_hash = $1
```

Comment:
We need to include bridging as well. For now at least deposits. Deposits are neither from or to the address, but they transfer funds to it.
Here's a sample operation like this: https://kaolin.golem.neti-soft.dev/tx/0x82b5885b108d6d75a90b65366bdc02c1dd6b56f510c48e5dbaaf549af1c67e76?tab=index. It's a deposit to address `0xDd329f6EDf93637634E8862d34c3909d298A7055`.

Maybe we can include internal transactions in the query? https://kaolin.golem.neti-soft.dev/tx/0x82b5885b108d6d75a90b65366bdc02c1dd6b56f510c48e5dbaaf549af1c67e76?tab=internal